### PR TITLE
docs: add workaround for ServiceCIDR setting for windows

### PR DESCRIPTION
**Pull request by jzhn**
Related #4088 

**Description**

**How was this change tested?**

Test setup:

- EKS v1.27.4-eks-2d98532
- Karpenter v0.30.0
- EKS Windows AMI `Windows_Server-2019-English-Core-EKS_Optimized-1.27-2023.08.17`
- VPC IP Range: `10.x.0.0/16` 
- Service IP Range: `10.100.0.0/16` (not overlapping with VPC)

Before workaround: `172.20.0.0/16` is set as ServiceCIDR
After workaround: `10.100.0.0/16` is set as ServiceCIDR

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jonathan-innis — DISMISSED** [review]
LGTM 🚀

File: website/content/en/docs/concepts/node-templates.md [hunk 1]
```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
 </powershell>
 ```
 
+{{% alert title="Note" color="primary" %}}
```
Comment (tzifudzi) [inline website/content/en/docs/concepts/node-templates.md:586]: To be consistent with other Windows docs, please set the title to 'Windows Support Notice' and the color to warning

File: website/content/en/docs/concepts/node-templates.md [hunk 2]
```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
...
+Currently, Karpenter does not specify `-ServiceCIDR` to [EKS Windows AMI Bootstrap script](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html#bootstrap-script-configuration-parameters).
+Windows worker nodes will use `172.20.0.0/16` or `10.100.0.0/16` for Kubernetes service IP address ranges based on the IP address of the primary interface.
+
+The effective ServiceCIDR can be verified at `$env:ProgramData\Amazon\EKS\cni\config\vpc-bridge.conf` on the worker node.
+If the effective ServiceCIDR is incorrect for your windows worker nodes, you can add below userData as workaround.
```
Comment (tzifudzi) [inline website/content/en/docs/concepts/node-templates.md:591]: Looks good. Please add the link to the Github issue for ServiceCIDR and add a statement that the feature is coming soon, but this is the workaround for now.

**Comment by jonathan-innis** [comment]
@jzhn When you make the changes from the review, can you make sure to propagate the changes to all the versions that were added to this PR so the change propagates across versions? Thanks!

File: website/content/en/docs/concepts/node-templates.md (hunk 1 above)
Comment (jzhn) [reply website/content/en/docs/concepts/node-templates.md]: thanks, updated.

File: website/content/en/docs/concepts/node-templates.md (hunk 2 above)
Comment (jzhn) [reply website/content/en/docs/concepts/node-templates.md]: Updated to mention the ongoing feature issue.

File: website/content/en/docs/concepts/node-templates.md [hunk 3]
```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
 </powershell>
 ```
 
+{{% alert title="Windows Support Notice" color="warning" %}}
+Currently, Karpenter does not specify `-ServiceCIDR` to [EKS Windows AMI Bootstrap script](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html#bootstrap-script-configuration-parameters).
+Windows worker nodes will use `172.20.0.0/16` or `10.100.0.0/16` for Kubernetes service IP address ranges based on the IP address of the primary interface.
+The effective ServiceCIDR can be verified at `$env:ProgramData\Amazon\EKS\cni\config\vpc-bridge.conf` on the worker node.
+
+Before [Karpenter ServiceCIDR support](https://github.com/aws/karpenter/issues/4088) is added, if the effective ServiceCIDR is incorrect for your windows worker nodes, you can add below userData as workaround.
```
Comment (jonathan-innis) [inline website/content/en/docs/concepts/node-templates.md:591]:
```suggestion
Support for the Windows ServiceCIDR argument can be tracked in a [Karpenter Github Issue](https://github.com/aws/karpenter/issues/4088). Currently, if the effective ServiceCIDR is incorrect for your windows worker nodes, you can add the following userData as workaround. 
```

Comment (jzhn) [reply website/content/en/docs/concepts/node-templates.md]: updated and synced to all related versions.

File: website/content/en/docs/concepts/node-templates.md [hunk 4]
```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
 </powershell>
 ```
 
+{{% alert title="Windows Support Notice" color="warning" %}}
+Currently, Karpenter does not specify `-ServiceCIDR` to [EKS Windows AMI Bootstrap script](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html#bootstrap-script-configuration-parameters).
+Windows worker nodes will use `172.20.0.0/16` or `10.100.0.0/16` for Kubernetes service IP address ranges based on the IP address of the primary interface.
+The effective ServiceCIDR can be verified at `$env:ProgramData\Amazon\EKS\cni\config\vpc-bridge.conf` on the worker node.
+
+Support for the Windows ServiceCIDR argument can be tracked in a [Karpenter Github Issue](https://github.com/aws/karpenter/issues/4088). Currently, if the effective ServiceCIDR is incorrect for your windows worker nodes, you can add the following userData as workaround.
```
Comment (jonathan-innis) [inline website/content/en/docs/concepts/node-templates.md:591]:
```suggestion
Support for the Windows ServiceCIDR argument can be tracked in a [Karpenter Github Issue](https://github.com/aws/karpenter/issues/4088). Currently, if the effective ServiceCIDR is incorrect for your windows worker nodes, you can add the following userData as a workaround.
```

nit: grammar

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Comment by tzifudzi** [comment]
Thanks for this workaround @jzhn

**Pull request merged**